.github/workflows/ci.yml

```diff
@@ -88,6 +88,9 @@ jobs:
       - name: Docker compose - integration tests
         if: ${{ matrix.testDockerCompose }}
         run: go run . integration-test
+        env:
+          # TODO(antlai-temporal): Remove this flag once server 1.26.2 released.
+          DISABLE_DEPLOYMENT_TESTS: "1"
```

Comment:
We should not merge unless/until we can remove this flag IMO. If that means we need an RC release of CLI we can, though hopefully CLI for 1.26 is released before this so users of this SDK can test too.

Reply:
We will have an RC for CLI and run  integration test in CI before merging, see: 
https://github.com/temporalio/sdk-go/blob/7e976b2a877e1fd95c98750a8a0e955989e0fd4f/internal/cmd/build/main.go#L143-L145

This flag only disables the tests for the docker image, which unfortunately only gets built for real releases, but the integration tests do run in against that CLI with -dev-server:
https://github.com/temporalio/sdk-go/blob/7e976b2a877e1fd95c98750a8a0e955989e0fd4f/.github/workflows/ci.yml#L47-L59

Reply:
Works for me. I wonder it the 1.26 release (and subsequent CLI release) will come before the RC.